### ISSUE TITLE
Login events: failed + success

### ISSUE THREAD

**Issue by Idontker** [amosproj/amos2022ss04-digital-identity#67]
## User story

1. As a {HR-employee}
2. I want / need {to get a response if my login process succeeded}
3. So that {I get redirected to the home-page or can try to login with my credentials again}

## Acceptance criteria

* Backend: set dummy password on creating a new DI to the data of the new DI
* Frontend: when "Login" is clicked: reroute to home-/dashboard-page or display failed-text using response data from backend

## Definition of done (DoD)

* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria

* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release

**Comment by Idontker** [comment]
Finished in sprint 5: 8b9cdf974b0592bb76a4d87a3d35fe9834c38272
